airbyte-integrations/connectors/destination-aws-datalake/build.gradle

```diff
@@ -2,7 +2,6 @@ plugins {
     id 'application'
     id 'airbyte-python'
     id 'airbyte-docker'
-    id 'airbyte-integration-test-java'
```

Comment:
FYI, I removed the java references because this is not a java connector.

Reply:
Good call.